packages/api/src/database/migrations/15_praise_allowed_channels_settings.ts

```diff
@@ -0,0 +1,43 @@
+import { SettingGroup } from '@settings/types';
+import { SettingsModel } from '../../settings/entities';
+
+const settings = [
+  {
+    key: 'PRAISE_ALLOWED_IN_ALL_CHANNELS',
+    value: true,
+    type: 'Boolean',
+    label: 'Praising allowed in all Channels',
+    description:
+      'If enabled, this allows praise to be dished in all channels in the server',
+    group: SettingGroup.DISCORD,
+  },
+  {
+    key: 'PRAISE_ALLOWED_CHANNEL_IDS',
+    value: '0, 0',
+    type: 'IntegerList',
```

Comment:
I'm unable to save a list of ids. 

The following should work, right? `984027559102677022, 945797725453160491`

<img width="315" alt="Skärmavbild 2022-06-20 kl  13 08 49" src="https://user-images.githubusercontent.com/9698363/174589627-4e0ace1d-037c-4b6c-b01a-c1c052c5d31a.png">